_posts/2021-12-27-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,346 @@
+---
+layout: post
+title:  "Trino 2021 Wrapped: A Year of Growth"
+author: Brian Olsen
+excerpt_separator: <!--more-->
+---
+
+As we reflect on Trino’s 2021 wrapped, one thing that stands out is the 
+tremendous growth we’ve seen this year in comparison to previous years. Yes,
+this is what all these year-in-retrospect blogs say, but this has some special 
+significance to it. Today marks the one-year anniversary since the project 
+[dropped the Presto name and moved to the Trino name](https://trino.io/blog/2020/12/27/announcing-trino.html).
+
+## Renaming from PrestoSQL to Trino
+
+This was a gut wrenching decision for many in the community who had grown with
+the project from early days. It also meant that the community would have to
+build affinity towards the new name from scratch. We weren't sure what to expect
+when the rename blog came out. What followed, was an outpouring of support and
+shock from the larger developer community. 
+
+<p align="center">
+ <a href="https://twitter.com/trinodb/status/1343330429684703232?s=20" target="_blank">
+   <img align="center" width="100%" src="/assets/blog/2021-review/tweets.png"/>
+ </a>
+</p>
+
+<!--more-->
+
+The community immediately got to work. The project had to change the namespace
+practically overnight from the `io.prestosql` namespace to `io.trino`, 
+a [migration blog](https://trino.io/blog/2021/01/04/migrating-from-prestosql-to-trino.html)
+was written, and we reached out to multiple projects to update compatibility to
+Trino.
+
+* [DBeaver](https://github.com/dbeaver/dbeaver/pull/10925)
+* [QueryBook](https://github.com/pinterest/querybook/issues/509)
+* [Homebrew](https://github.com/Homebrew/homebrew-core/pull/83185)
+* [dbt](https://github.com/dbt-labs/dbt-presto/issues/39)
+* [sqlalchemy](https://github.com/dungdm93/sqlalchemy-trino/issues/20)
+* [sqlpad](https://github.com/sqlpad/sqlpad/pull/974)
+* [Apache Superset](https://github.com/apache/superset/pull/13105)
+* [Redash](https://github.com/getredash/redash/pull/5411)
+* [Awesome Java](https://github.com/akullpp/awesome-java/pull/917)
+* [Awesome For Beginners](https://github.com/MunGell/awesome-for-beginners/pull/933)
+* [Airflow](https://github.com/apache/airflow/pull/15187)
+* [trino-gateway](https://github.com/lyft/presto-gateway/issues/134)
+* [Metabase](https://github.com/metabase/metabase/issues/17532)
+* and so much more...
+
+Not only was the community excited and supportive about the brand change, but
+particularly they were all loving the new bunny mascot, 
```

Comment:
the new mascot. Our adorable bunny was soon after named Commander Bun Bun by the community.